Composer/ScriptHandler.php

```diff
@@ -459,30 +425,10 @@ protected static function getConsoleDir(CommandEvent $event, $actionName)
     {
         $options = static::getOptions($event);
 
-        if (static::useNewDirectoryStructure($options)) {
-            if (!static::hasDirectory($event, 'symfony-bin-dir', $options['symfony-bin-dir'], $actionName)) {
-                return;
-            }
-
-            return $options['symfony-bin-dir'];
-        }
-
         if (!static::hasDirectory($event, 'symfony-app-dir', $options['symfony-app-dir'], 'execute command')) {
             return;
         }
 
         return $options['symfony-app-dir'];
     }
```

Comment:
hmm, this would break things for people using a project with the new structure and using ``symfony-bin-dir`` to specify the location of the console (I know some people do it). We should keep support for the explicit configuration of the console dir IMO